bagit.py

```diff
@@ -648,7 +655,7 @@ def _calculate_file_hashes(full_path, f_hashers):
 
 
 def _load_tag_file(tag_file_name):
-    with open(tag_file_name, 'r') as tag_file:
+    with open_text_file(tag_file_name, 'r', encoding='utf-8-sig') as tag_file:
```

Comment:
same issue here, need to open with encoding specified in bagit.txt which may or may not be utf-8